src/components/Wrapper/index.js

```diff
@@ -1,12 +1,15 @@
 import classNames from 'classnames';
+import routeMap from '../../utils/routeMap';
+
 
 import wrapperStyle from './style.scss';
 
 const Wrapper = ({
-  className, children, id, isHome,
+  className, children, id, isHome, location,
```

Comment:
I would prefer the `isWhite` prop passed here.

Reply:
👍